src/Arcus.EventGrid/Parsers/EventParser.cs

```diff
@@ -107,5 +106,11 @@ internal static Event ParseJObject(JObject rawInput)
                 return eventGridEvent;
             }
         }
+
+        private static bool IsCloudEvent(JObject jObject)
```

Comment:
Idea - Provide unit tests for this or is it covered somewhere?

Reply:
It's actually from the SDK.

Reply:
Even then, shouldn't we provide testing as it's pretty crucial?

For example, do we have unit tests where we have all 4 flavors (CE 0.1, CE 0.2, CE 1.0 & EG) and verify they are parsed correctly?

Reply:
Seperate PR is ok

Reply:
The EventGrid only allows the v1.0 CloudEvent schema, so don't even sure if we should allow others.

Reply:
Existing endpoints before 1.0 still support 0.1 & 0.2

Reply:
#126